docs/spi_driver.md

```diff
@@ -62,7 +83,7 @@ Write a byte to the selected SPI device.
  - `uint8_t data`  
    The byte to write.
  - `uint16_t timeout`  
-   The amount of time to wait, in milliseconds, before timing out.
+   The amount of time to wait, in milliseconds, before timing out. This value is ignored for ARM.
```

Comment:
If the timeouts are ignored for ARM, should we just do away with them altogether? I was only copying the I2C signatures.

Reply:
Realistically I don't see that they really add value -- SPI is clocked by the master, and if the slave isn't responding in time you're going to get zero's regardless of AVR or ARM.

Reply:
The responses come back by sending bytes to the slave anyway, as it's a synchronous protocol. If it's not responding while sending those bytes, it's not going to be timeouts that are the problem.

Reply:
Feel free to remove them then.

Reply:
Done.